apps/meteor/client/views/admin/apps/AppDetailsPage/AppDetailsPageHeader.tsx

```diff
@@ -1,18 +1,32 @@
 import type { App } from '@rocket.chat/core-typings';
-import { Box } from '@rocket.chat/fuselage';
+import { Box, Tag, Tooltip } from '@rocket.chat/fuselage';
 import { useTranslation } from '@rocket.chat/ui-contexts';
 import moment from 'moment';
 import React, { ReactElement } from 'react';
 
 import AppAvatar from '../../../../components/avatar/AppAvatar';
 import AppMenu from '../AppMenu';
 import BundleChips from '../BundleChips';
+import { appIncompatibleStatusProps } from '../helpers';
 import AppStatus from './tabs/AppStatus';
+import { TooltipOnHover } from './tabs/AppStatus/TooltipOnHover';
+
+const versioni18nKey = (app: App): string => {
+	const { version, marketplaceVersion, marketplace } = app;
+	const marketplaceCast = marketplace as boolean;
+
+	if (marketplaceCast) {
```

Comment:
Why not check if `typeof marketplace === 'boolean' instead of casting? Casting it with typescript doesn't validate it is actually a boolean, it only convinces typescript it is a boolean and then you check if it is falsey.

Reply:
Make sense! Fixed ✅ Thanks 👊